packages/wallet/api/src/lib-wallet.js

```diff
@@ -1528,6 +1528,23 @@ export function makeWallet({
     return offerResult.uiNotifier;
   }
 
+  /**
+   * Gets the notifiers from an offer's result.
+   *
+   * @param {string} rawId - The offer's raw id.
+   * @param {string} dappOrigin - The origin of the dapp the offer came from.
+   */
+  async function getNotifiers(rawId, dappOrigin = 'unknown') {
+    const id = makeId(dappOrigin, rawId);
+    const offerResult = await idToOfferResultPromiseKit.get(id).promise;
+    assert(
+      passStyleOf(offerResult) === 'copyRecord',
+      `offerResult must be a record to have notifiers`,
+    );
+    assert(offerResult.notifiers, X`offerResult does not have notifiers`);
+    return offerResult.notifiers;
```

Comment:
It's important to ensure the notifiers are a record as well.

As a general practice, please only ask for `offerResult.notifiers` once (via destructuring).  It's not as important here, but we want to encourage programmers to "avoid conflicting answers by only asking orthogonal questions".

```suggestion
    const { notifiers } = offerResult;
    assert(notifiers, X`offerResult does not have notifiers`);
    assert(
      passStyleOf(notifiers) === 'copyRecord',
      X`notifiers ${notifiers} must be a record`,
    );
    return notifiers;
```

Reply:
Done